### PR TITLE
docs: add link color doc

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      -     |
| 🇨🇳 Chinese |    -       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: docs/react/customize-theme.zh-CN.md [hunk 1]
```diff
@@ -348,6 +348,10 @@ export default () => {
 
 在 ConfigProvider 中我们通过 `DesignTokenContext` 传递 context，`theme` 为 `undefined` 时不会套一层 Provider，所以从无到有或者从有到无时 React 的 VirtualDOM 结构变化，导致组件重新 mount。解决方法：将 `undefined` 替换为空对象 `{}` 即可。
 
+### 为什么修改 `colorPrimary` 不会改变 `link` 类型的 Button 和 Typography.Link 的颜色？
+
+在 5.0 中，我们让 `colorLink` 跟随了 `colorInfo`，原因是在大部分场景下我们都希望链接和浏览器原生的一样是蓝色的，不跟随 `colorPrimary` 变化。但您仍然可以通过修改 `colorLink` 和 `colorLinkHover` 来覆盖这一默认取值。
```
Comment (afc163) [inline docs/react/customize-theme.zh-CN.md:353]: colorLinkHover 应该不用改，自动算就行了

Comment (MadCcc) [reply docs/react/customize-theme.zh-CN.md:353]: colorLinkHover 取的是 colorInfo 色板里的 colorInfoHover，aliasToken 现在没有这种算法。

Comment (afc163) [reply docs/react/customize-theme.zh-CN.md:353]: 这样不对，colorLinkHover 应该跟着 colorLink 走

Comment (afc163) [reply docs/react/customize-theme.zh-CN.md:353]: 否则没办法单独修改 colorLink

Comment (arvinxx) [reply docs/react/customize-theme.zh-CN.md:353]: > 否则没办法单独修改 colorLink

如果按之前讨论的结论，colorLink 不应该单独改，应该跟着 colorInfo 改。

**Pull request closed without merging**